resources/views/admin.blade.php

```diff
@@ -169,11 +169,11 @@ function deleteEmployee(id){
                         $('#email').val(data.email)
                         $('#phone').val(data.phone)
                         $('#address').val(data.address)
-                        $('#select').append($('<option>', {
-                            value: data.company.id,
-                            text: data.company.name,
-                            selected: "selected",
-                        }))
+                        $("#select option").each(function(){
```

Comment:
Don't loop over the option. this is not a best practice. we have tables in live system where there are 22K data exist if we loop it will crash the application on run time. looping through the option should be avoided. please find a different approach.

ref
![image](https://github.com/user-attachments/assets/f9f37d50-c2ba-46a2-bd38-3e9e141aab94)

Reply:
Sure i can understand ill find a different approach without looping

Reply:
i have fixed the issue by selecting the id of the option please review @Raza9798 #33